tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/swtbot/PitResultNotifier.java

```diff
@@ -17,16 +17,13 @@
 package org.pitest.pitclipse.ui.swtbot;
 
 import org.pitest.pitclipse.core.extension.point.ResultNotifier;
-import org.pitest.pitclipse.ui.PitclipseTestActivator;
 import org.pitest.pitclipse.ui.extension.point.PitUiUpdate;
 import org.pitest.pitclipse.ui.swtbot.ResultsParser.Summary;
 
 public class PitResultNotifier implements ResultNotifier<PitUiUpdate> {
     @Override
     public void handleResults(PitUiUpdate updateEvent) {
-        if (testsAreInProgress()) {
-            notifiyTestsOfHtmlResults(updateEvent);
-        }
+        notifiyTestsOfHtmlResults(updateEvent);
```

Comment:
Are you sure we can safely get rid of the `if` check? I don't remember well why it was here in the first place but I guess that `notifyTestsOfHtmlResults` was supposed to be called only during Pitclipse's UI tests.

Reply:
We're inside the UI tests project anyway so that's always the case.
The flag was set by the old test runner for Cucumber that's not used anymore.